.changeset/nine-turtles-learn.md

```diff
@@ -0,0 +1,29 @@
+---
+"@logto/experience": patch
+"@logto/core": patch
+---
+
+## Fix the new user from SSO register hook event not triggering bug
```

Comment:
the first line should be a sentence starting with lowercase, thus it can be shown as a summary in the changelog

```suggestion
fix the new user from SSO register hook event not triggering bug
```